hera/editor/MapEditorExtraCursors.tsx

```diff
@@ -0,0 +1,35 @@
+import Vector from '@deities/athena/map/Vector.tsx';
+import { SizeVector } from '@deities/athena/MapData.tsx';
+import { ComponentProps, memo } from 'react';
+import Cursor from '../Cursor.tsx';
+import getSymmetricPositions from './lib/getSymmetricPositions.ts';
+import { DrawingMode } from './Types.tsx';
+
+export default memo(function MapEditorExtraCursors({
+  defaultCursorPosition,
+  drawingMode,
+  mapSize,
+  ...props
+}: {
+  defaultCursorPosition: Vector | null;
+  drawingMode: DrawingMode | undefined;
+  mapSize: SizeVector;
+} & Omit<ComponentProps<typeof Cursor>, 'position'>) {
+  if (!defaultCursorPosition || !drawingMode || drawingMode === 'regular') {
+    return null;
+  }
+
+  const vectors = getSymmetricPositions(
+    defaultCursorPosition,
+    drawingMode,
+    mapSize,
+  );
+
+  if (!vectors.length) {
+    return null;
+  }
+
+  return vectors.map((vector) => (
+    <Cursor key={vector.toString()} {...props} position={vector} />
```

Comment:
You can just pass `vector`, `toString` is implicitly called.

Reply:
TypeScript doesn't like that: 
```Type 'Vector' is not assignable to type 'Key | null | undefined'.ts(2322)```